proto/accessgraph/v1alpha/access_graph_service.proto

```diff
@@ -74,39 +48,24 @@ message QueryRequest {
 
 // QueryResponse is a response to a query.
 message QueryResponse {
-  repeated Node nodes = 1;
-  repeated Edge edges = 2;
+  // nodes is a list of nodes returned by the query. The response may contain only nodes.
+  repeated accessgraph.v1alpha.Node nodes = 1;
+  // edges is a list of edges returned by the query.
+  repeated accessgraph.v1alpha.Edge edges = 2;
```

Comment:
You should be able to use the short-hand here, it's the same package after all.

```suggestion
  repeated Node nodes = 1;
  // edges is a list of edges returned by the query.
  repeated Edge edges = 2;
```

Reply:
Works, thanks